src/state/actions/codeSystemActions.ts

```diff
@@ -0,0 +1,32 @@
+import { CodeSystem } from "../../resources/ts/proto/r4/core/resources/code_system_pb";
+
+export const GET_CODESYSTEMDATATYPE_SUCCESS = "GET_CODESYSTEMDATATYPE_SUCCESS";
+export const GET_CODESYSTEMDATATYPE_FAILURE = "GET_CODESYSTEMDATATYPE_FAILURE";
```

Comment:
Naming your const like this makes them difficult to read:
```suggestion
export const GET_CODE_SYSTEM_DATA_TYPE_SUCCESS = "GET_CODESYSTEMDATATYPE_SUCCESS";
export const GET_CODE_SYSTEM_DATA_TYPE_FAILURE = "GET_CODESYSTEMDATATYPE_FAILURE";
```